batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,169 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.services.EmployeeService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class EmployeeController
+ */
+
+@WebServlet("/employees/*")
+public class EmployeeController extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+
+    EmployeeService employeeService = new EmployeeService();
+    Employee emp = new Employee();
+
+    public static String employeeListUrl = "/batch7crud-roll5/employees";
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String path = request.getPathInfo();
+        if (path != null) {
+            String[] pathParts = path.split("/");
+
+            if ("create".equals(pathParts[1]))
+                create(request, response);
+
+            else if ("edit".equals(pathParts[2])) {
+                int id = Integer.parseInt(pathParts[1]);
+                edit(request, response, id);
+            }
+
+            else if ("page".equals(pathParts[1])) {
+                int pageNo = Integer.parseInt(pathParts[2]);
+                fetchData(request, response, pageNo - 1);
+            }
+
+        }
+
+        else {
+            int pageNo = 1;
+            fetchData(request, response, pageNo - 1);
+        }
+
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String path = request.getPathInfo();
+        if (path != null) {
+            String[] pathParts = path.split("/");
+
+            if ("createProcess".equals(pathParts[1])) {
+                createProcess(request, response);
+
+            }
+
+            else if ("editProcess".equals(pathParts[2])) {
+                int id = Integer.parseInt(pathParts[1]);
+                editProcess(request, response, id);
+
+            }
+
+            else if ("deleteProcess".equals(pathParts[2])) {
+                int id = Integer.parseInt(pathParts[1]);
+                deleteProcess(request, response, id);
+            }
+
+        } else {
+            int pageNo = 1;
+            fetchData(request, response, pageNo - 1);
+        }
+
+    }
+
+    private void deleteProcess(HttpServletRequest request, HttpServletResponse response, int id) {
+        try {
+            employeeService.deleteById(id);
+
+            response.sendRedirect(employeeListUrl);
+
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)